src/containeirs/NoMatch/NoMatch.tsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
-import styled from 'styled-components';
+import { Button, ButtonType } from 'components/Button';
+import { Link } from 'react-router-dom';
+import {
+  Wrapper, LeftSide, RightSide, NoMatchAltLoader, NoMatchLoader, NoMatchLogo, NoMatchAppTitle, NoMatchText, LogoNameContainer, RedirectButton,
+} from './styled';
 
-const Wrapper = styled('div')`
-  width: 80%;
-  float:right;
-`;
-
-const NoMatchPage = () => {
+const NoMatch = () => {
   return (
     <Wrapper>
-      <p>No Match</p>
+      <LeftSide>
```

Comment:
Te nowe liniie usunąć. Nigdy nie robimy czegoś takiego bo zmniejsza to czytelność kodu

Reply:
A sory. Błąd rendrowania xD